ext/ffi/lib.rs

```diff
@@ -374,3 +368,41 @@ fn op_ffi_call(
     }
   })
 }
+
+fn op_ffi_call(
+  state: &mut deno_core::OpState,
+  args: FfiCallArgs,
+  _: (),
+) -> Result<Value, AnyError> {
+  let resource = state
+    .resource_table
+    .get::<DynamicLibraryResource>(args.rid)?;
+
+  let symbol = resource
+    .symbols
+    .get(&args.symbol)
+    .ok_or_else(bad_resource_id)?;
+
+  ffi_call(args, symbol)
+}
+
+/// A non-blocking FFI call.
+async fn op_ffi_call_async(
+  state: Rc<RefCell<deno_core::OpState>>,
+  args: FfiCallArgs,
+  _: (),
+) -> Result<Value, AnyError> {
+  let resource = state
+    .borrow()
+    .resource_table
+    .get::<DynamicLibraryResource>(args.rid)?;
+  let symbols = &resource.symbols;
+  let symbol = symbols
+    .get(&args.symbol)
+    .ok_or_else(bad_resource_id)?
+    .clone();
+
+  tokio::task::spawn_blocking(move || ffi_call(args, &symbol))
+    .await
+    .unwrap()
```

Comment:
Rather than `op_ffi_call_async` I think it should be a bit more specific `op_ffi_call_nonblocking` for example.

Reply:
Sure, done.

Reply:
I disagree, op_ffi_call_async was perfectly apt.
`op_ffi_call_nonblocking` tells me that the FFI call returns EAGAIN every now and then.